src/hmrc-design-patterns/service-timeout/index.njk

```diff
@@ -0,0 +1,200 @@
+---
+title: Service timeout
+menuText: Service timeout
+status: experimental
+layout: twoColumnPage.njk
+---
+{% from "_example.njk" import example %}
+{% from "_codeSnippet.njk" import codeSnippet %}
+
+{% block content %}
+
+<p class="govuk-body">This pattern warns the user before we time them out of a service and tells them what has happened when we do time them out.</p>
+
+<h2 class="govuk-heading-l" id="when-to-use">When to use</h2>
+
+<p class="govuk-body">We time the user out when they do not do anything for 15 minutes. Warn them after 13 minutes, which gives them 2 minutes to decide to keep using the service.</p>
+
+<h2 class="govuk-heading-l" id="how-it-works">How it works</h2>
+
+<p class="govuk-body">Use an accessible dialog box that appears on top of the page. This stops them using the screen with a mouse or keyboard until they select an option or press the escape key.</p>
+
+<p class="govuk-body">The user can:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>select a button to stay in the service and not time out</li>
+  <li>select a link to sign out or have their answers deleted</li>
+  <li>press the escape key to close the warning, stay in the service and not time out</li>
+  <li>do nothing</li>
+</ul>
+
+  <h3 class="govuk-heading-m" id="when-the-user-is-signed-in">When the user is signed in</h3>
+
+  <p class="govuk-body">The warning should:</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>say ‘For your security, we will sign you out in 2 minutes.’</li>
+    <li>say what will happen to their answers, if needed</li>
+    <li>have a ‘Stay signed in’ button</li>
+    <li>have a ‘Sign out’ link that signs them out as normal</li>
+  </ul>
+
+  {{ example({
+    item: 'service-timeout',
+    example: 'default',
+    description: 'Service timeout default',
+    height: 400
+  }) }}
+
+  {{ example({
+    item: 'service-timeout',
+    example: 'saved',
+    description: 'Saved service timeout',
+    height: 400
+  }) }}
+
+  {{ example({
+    item: 'service-timeout',
+    example: 'not-saved',
+    description: 'Not saved service timeout',
+    height: 400
+  }) }}
+
+  <p class="govuk-body">If they do nothing, take them to a ‘We signed you out’ page that uses similar content as the warning box.</p>
+
+  {{ example({
+    item: 'service-timeout',
+    example: 'signed-out-default',
+    description: 'Signed out timeout default'
+  }) }}
+
+  {{ example({
```

Comment:
Visually these three examples look identical - are they supposed to be different?